backend/internal/server/apis/sports/components/logo.go

```diff
@@ -39,13 +39,13 @@ func handleLogoUpload(c *fiber.Ctx) error {
 	logo := models.Logo{
 		Image: imageBuffer,
 	}
-	err = db.SaveLogo(&logo)
+	record, err := db.SaveLogo(&logo)
 	if err != nil {
 		log.WithErr(err).Alert("Failed to save logo")
 		return responder.InternalServerError(c)
 	}
 
-	return responder.Ok(c)
+	return responder.CreatedWithData(c, record)
```

Comment:
This one I'm not sure what it's returning but we needto make sure we aren't returning the full record but more-so just the ID that they can use to fetch this.

Reply:
For this API specifically that makes a lot of sense